lib/uglifier.js

```diff
@@ -23,7 +23,7 @@ const usedCacheKeys = [];
 
 function processAssets(compilation, options) {
   const assetHash = compilation.assets;
-  const useSourceMaps = !!compilation.options.devtool;
+  const useSourceMaps = !!(options.uglifyJS && options.uglifyJS.sourceMap);
```

Comment:
Can you move this up to the options object? Uglify2 doesn't use this option, and v3 does but it expects an object.  I'd like for this to remain a true/false boolean on the options object, that way the uglifyJS object can continue to be passed straight to uglify.

Also, could you add an entry to the README for `sourceMap`?